sdk/resource/builtin.go

```diff
@@ -78,7 +79,8 @@ func StringDetector(schemaURL string, k attribute.Key, f func() (string, error))
 	return stringDetector{schemaURL: schemaURL, K: k, F: f}
 }
 
-// Detect implements Detector.
+// Detect returns a *Resource that describes the string as a value
+// corresponding to k as well as the specific schemaURL.
```

Comment:
What does `k` refer to here?

Reply:
It stands for `attribute.Key` .

Reply:
Please update the comment here to make it clear.